lib/s3routes/routesUtils.js

```diff
@@ -1,6 +1,11 @@
 const url = require('url');
 const ipCheck = require('../ipCheck');
 const errors = require('../errors');
+const { eachSeries } = require('async');
+
+const responseErr = new Error();
```

Comment:
I would suggest using one of the ArsenalError duplicated with `.customizeDescription('...')`

Reply:
This should be ok since the error is not sent to the client but we use it internally in the waterfall.